src/modules/ROOT/pages/changelog/202209/03.json

```diff
@@ -0,0 +1,139 @@
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/18248a2376ec4400ac562faae32be3ec129acca1"
+      ]
+    },
+    {
+      "what": [
+        "new filter operators (`IN` and `NOT IN`) to the pre-filter in the reporter UI.",
+        "This why users have the ability to show or not show certain value contained in the record."
+      ],
+      "why": [
+        "because to enable the users to have the option to use `IN/NOT IN` operator in Reporter's filter for better control of what they want to show on the record lists."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/560/commits/22ceca9ad21bbd9f2675930879128030696d4c8f"
+      ]
+    },
+    {
+      "what": [
+        "extended configuration for pie, bar, line, and doughnut charts.",
+        "Three new sections were added - tooltip, legend and offset as well as the options to rotate x and y axis labels (only for bar and line charts)."
+      ],
+      "why": [
+        "to give the users more freedom to customize charts."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/a8d3b831db94538f6cfd84bd86fe35ec555b1bc7",
+        "https://github.com/cortezaproject/corteza/commit/b2c9de57a73c84aa16f255e8f7c281be91c76c34"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "the resource list now uses the same component, item count and pagination and it is also enabled for resource lists that support it.",
+        "Now almost on every resource we show the total item count and we have the ability to go to the next page if there are items."
+      ],
+      "why": [
+        "because the look and feel of lists across {PRODUCT_NAME} needed to be unified.",
+        "Also for better user experience and to always know the correct number of how many records/items we have."
+      ],
+      "how": [
+        "by updating and using the reusable CResourceList component in corteza-vue.",
+        "That means we can control the look of all lists from one place."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-vue/commit/030109b86e3e2d35ec0f0104f5b279c0f33f7ebd", 
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/229bf12d9c6608f40bf49dfb6ef4ca68defa0875", 
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/c67daa7a60e1fb38a49a0df770aab6257e5f03bb", 
+        "https://github.com/cortezaproject/corteza-webapp-reporter/commit/06c07d8036f8ad3736c06bd69ae1ab23df095563",
+        "https://github.com/cortezaproject/corteza-webapp-workflow/commit/60334336292aca9ed4f581213d6d4bb300772986"
+      ]
+    },
+    {
+      "what": [
+        "the requirement to provide a handle when creating a resource is removed and this is now optional.",
+        "The change is applied in corteza-webapp-compose, corteza-webapp-admin, corteza-webapp-reporter and corteza-webapp-workflow."
```

Comment:
You missed my comment here:


Shouldn't we use the new mono-repo _notations_ here? Perhaps you could simply list out the web application names.

Also; `...corteza-webapp-reporter and corteza-webapp-workflow...` -> `...corteza-webapp-reporter, and corteza-webapp-workflow...`